README.md

```diff
@@ -21,6 +21,9 @@
 
 ------
 
+## Build
```

Comment:
I think we shouldn't add internal links to upstream README

Reply:
but how can we link our build to the project to make it convenient?

Reply:
I'm not sure if we are going to contribute to upstream

Reply:
removed